eval/eval.go

```diff
@@ -36,13 +36,15 @@ const (
 const (
 	defaultValuePrefix = "▶ "
 	initIndent         = vals.NoPretty
+	defaultSuppressBackground = "false"
 )
 
 // Evaler is used to evaluate elvish sources. It maintains runtime context
 // shared among all evalCtx instances.
 type Evaler struct {
 	evalerScopes
 	valuePrefix  string
+	suppressBackground string
```

Comment:
Use a `bool`.

Prefer using positive-meaning names for options. In this case, call the field `notifyBgJobSuccess`, give it a default value of `true`, and call the Elvish variable `$notify-bg-job-success`